### ISSUE TITLE
Rename the module

### ISSUE THREAD

**Issue by mscherer** [OSAS/ansible-role-middleman_builder#4]
As there is some community who may want to build more than middleman, it might make sense to rename the module to be static_builder rather than middleman_builder.

So we need to have a list of downstream user we know and be prepared to switch.

**Comment by duck-rh** [comment]
Here I have: Community / oVirt / RDO / OSCI / Gluster
(MI no more uses it)
Anything else?

**Comment by mscherer** [comment]
Nope. I think we can just rename and fix. The failure would either be quite visible anyway, or wouldn't break anything but deployment of infra, so no userfacing breakage.

**Comment by duck-rh** [comment]
Done, let's fix now.

**Comment by duck-rh** [comment]
It would be nice to change the default _builder_username_ too, but this means renaming things on the deployed machines too.

**Comment by duck-rh** [comment]
I fixed all of them except Gluster because I don't have access so I made a PR.

**Comment by duck-rh** [comment]
Updated the doc slightly in #18

**Comment by mscherer** [comment]
We can just add a alias on the variable for builder_username.
